src/aks-preview/azext_aks_preview/tests/latest/test_validators.py

```diff
@@ -118,30 +118,30 @@ def __init__(self, api_server_authorized_ip_ranges=None, cluster_autoscaler_prof
         self.cluster_autoscaler_profile = cluster_autoscaler_profile
 
 
-class TestVNetSubnetId(unittest.TestCase):
-    def test_invalid_vnet_subnet_id(self):
+class TestSubnetId(unittest.TestCase):
```

Comment:
Is it possible to add ScenarioTest instead of UnitTest for this new parameter ?

Reply:
Honestly I'm reluctant since I don't feel the scenario tests are well documented and another coworker (bo wang)  just spent multiple days fixing them after needing to re-record for new api version. 
https://teams.microsoft.com/l/message/19:106ed06b2a3745b1a7ee5c573ab098c6@thread.skype/1605207074453?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=e121dbfd-0ec1-40ea-8af5-26075f6a731b&parentMessageId=1605207074453&teamName=Azure%20Container%20Compute&channelName=AKS%20API%20channel&createdTime=1605207074453

We have end to end tests on our part so I'm not sure a scenario test gets us much.